src/execution/__tests__/simplePubSub.ts

```diff
@@ -51,6 +51,7 @@ export class SimplePubSub<T> {
         emptyQueue();
         return Promise.resolve({ value: undefined, done: true });
       },
+      /* c8 ignore next 4 */
```

Comment:
why do we need to ignore these?

Reply:
Good call. We don't any more! I will check any other additional c8 ignores I added.